nevergrad/functions/olympussurfaces/core.py

```diff
@@ -0,0 +1,121 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Based on https://github.com/aspuru-guzik-group/olympus
+
+import numpy as np
+from functools import partial
+from nevergrad.parametrization import parameter as p
+from ..base import ExperimentFunction
+
+import nevergrad as ng
+
+try:
+    from olympus import surfaces  # type: ignore
+    from olympus import noises  # type: ignore
+    from olympus import Emulator  # type: ignore
+    from olympus.datasets import Dataset  # type: ignore
+except ImportError as e:
+    raise ng.errors.UnsupportedExperiment("Please install olympus for Olympus experiments") from e
+
+
+class OlympusSurface(ExperimentFunction):
+
+    traditional_surfaces = {
+        "Michalewicz": surfaces.Michalewicz,
+        "AckleyPath": surfaces.AckleyPath,
+        "Dejong": surfaces.Dejong,
+        "HyperEllipsoid": surfaces.HyperEllipsoid,
+        "Levy": surfaces.Levy,
+        "Michalewicz": surfaces.Michalewicz,
+        "Rastrigin": surfaces.Rastrigin,
+        "Rosenbrock": surfaces.Rosenbrock,
+        "Schwefel": surfaces.Schwefel,
+        "StyblinskiTang": surfaces.StyblinskiTang,
+        "Zakharov": surfaces.Zakharov,
+        "DiscreteAckley": surfaces.DiscreteAckley,
+        "DiscreteDoubleWell": surfaces.DiscreteDoubleWell,
+        "DiscreteMichalewicz": surfaces.DiscreteMichalewicz,
+        "LinearFunnel": surfaces.LinearFunnel,
+        "NarrowFunnel": surfaces.NarrowFunnel,
+        "GaussianMixture": surfaces.GaussianMixture,
+    }
```

Comment:
can this be created with just: `{name: getattr(surfaces, name) for name in dir(surfaces)}`?
There may be a few additional gotchas (getting only if it's the correct class, skipping some of the names which are not supported) but this way the code will be much more maintainable.

Or otherwise just create `names = ["Michalewicz", ...]` if you prefer to make sure we do not use other surfaces, then, the corresponding object is just `getattr(surfaces, name)`, so we can remove a lot of duplications